sdk/eventhubs/azure-messaging-eventhubs/src/main/java/com/azure/messaging/eventhubs/EventHubMessageSerializer.java

```diff
@@ -1,42 +1,41 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
-package com.azure.messaging.eventhubs.implementation;
+package com.azure.messaging.eventhubs;
 
 import com.azure.core.amqp.MessageConstant;
-import com.azure.core.implementation.util.ImplUtils;
-import com.azure.messaging.eventhubs.EventData;
+import com.azure.core.util.Context;
+import com.azure.core.util.logging.ClientLogger;
+import com.azure.messaging.eventhubs.implementation.ManagementChannel;
+import com.azure.messaging.eventhubs.implementation.MessageSerializer;
 import org.apache.qpid.proton.Proton;
 import org.apache.qpid.proton.amqp.Binary;
 import org.apache.qpid.proton.amqp.Symbol;
 import org.apache.qpid.proton.amqp.messaging.AmqpValue;
 import org.apache.qpid.proton.amqp.messaging.ApplicationProperties;
 import org.apache.qpid.proton.amqp.messaging.Data;
 import org.apache.qpid.proton.amqp.messaging.MessageAnnotations;
+import org.apache.qpid.proton.amqp.messaging.Section;
 import org.apache.qpid.proton.message.Message;
 
+import java.nio.ByteBuffer;
+import java.util.Date;
```

Comment:
Is it possible to move away from `java.util.Date`? IntelliJ keeps complaining about its usage as it's obsolete.

Reply:
It's a type that is serialized and returned from qpid-proton-j. We'd have to update the underlying library to use Instant if we wanted to remove this.

Reply:
FWIW, I don't mind using this internally, I just don't want us to use Date in our public APIs. This is a good candidate for a CheckStyle rule.